tests/java/resources/cardinality/atTestData.json

```diff
@@ -0,0 +1,105 @@
+{
```

Comment:
can you write description and link where this came from for all data copies from other repo or put these data into separate directory and put README

Reply:
can I add them as a git submodule?

Reply:
Sure

Reply:
done